File: CS/1015 - Distancia entre dois pontos.cs
using System;

class URI
{
    static void Main()
    {
        string[] first_numbers = Console.ReadLine().Split();
        string[] second_numbers = Console.ReadLine().Split();

        float x1 = float.Parse(first_numbers[0]);
        float y1 = float.Parse(first_numbers[1]);
        float x2 = float.Parse(second_numbers[0]);
        float y2 = float.Parse(second_numbers[1]);

        double dif = Math.Sqrt(Math.Pow((x2-x1), 2) + Math.Pow((y2-y1), 2));

        Console.WriteLine("{0}", dif.ToString("0.0000"));
    }
}

File: Java/1015 - Distancia entre dois pontos.java
import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        float x1 = input.nextFloat();
        float y1 = input.nextFloat();
        float x2 = input.nextFloat();
        float y2 = input.nextFloat();

        double dif = Math.sqrt(Math.pow((x2-x1), 2) + Math.pow((y2-y1), 2));
    
        System.out.printf("%.4f\n", dif);
    }
}